local-annotations-library-documentation.md

```diff
@@ -0,0 +1,83 @@
+# PaSh Setup and Execution Guide
```

Comment:
This file should be moved under `docs` and we should add a pointer sentence to it from the README :)